scss/_breadcrumb.scss

```diff
@@ -2,8 +2,6 @@
 .breadcrumb {
   display: flex;
   flex-wrap: wrap;
-  padding: $breadcrumb-padding-y $breadcrumb-padding-x;
-  margin-bottom: $breadcrumb-margin-bottom;
```

Comment:
Restore these lines.

Reply:
In `_custom-variables.scss`, modify the breadcrumbs section to look like
```scss
// >> Breadcrumbs
$breadcrumb-color: $azurite;
$breadcrumb-bg: $white;
$breadcrumb-padding-y: 0px;
$breadcrumb-padding-x: 0px;
$breadcrumb-margin-bottom: 0px;
```

Reply:
```
scss/_custom-variables.scss
 233:25  ✖  Unexpected unit  length-zero-no-unit
 234:25  ✖  Unexpected unit  length-zero-no-unit
 235:29  ✖  Unexpected unit  length-zero-no-unit
```